docusaurus.config.js

```diff
@@ -243,11 +243,6 @@ const config = {
         defaultMode: "light",
         disableSwitch: false,
         respectPrefersColorScheme: false,
-        /* we clear these icons out and use CSS to swap in our own images instead */
-        switchConfig: {
```

Comment:
Does this still work if you remove this config? Do we have to make this change somewhere else?

Reply:
This worked hand-in-hand with the swizzled Toggle Component. Now that we are no longer swizzling it; the config is not required.